plugins/autofarm.cpp

```diff
@@ -340,7 +341,7 @@ class AutoFarm {
 
         for (auto& th : thresholds)
         {
-            if (lastCounts[th.first] > 0)
+            if (lastCounts.find(th.first) != lastCounts.end())
```

Comment:
Another way to put this, which I have found more succinct, is `if (lastCounts.count(th.first))`

Reply:
that is potentially _slightly_ less efficient at the machine code level but not enough to care, especially in this context